OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -55,6 +55,8 @@ void Millard2012EquilibriumMuscle::constructProperties()
     constructProperty_ForceVelocityCurve(ForceVelocityCurve());
     constructProperty_FiberForceLengthCurve(FiberForceLengthCurve());
     constructProperty_TendonForceLengthCurve(TendonForceLengthCurve());
+
+    setMinControl(get_minimum_activation());
```

Comment:
IIUC, the `min_control` and `minimum_activation` properties could both be set during deserialization. It seems that the following constraints would need to be enforced post-deserialization (i.e., in `extendFinalizeFromProperties()`:

1. `LB <= minimum_activation <= 1`, where `LB` is 0 or 0.01 depending on Millardian flags
2. Let `userPropVal = get_min_control()`. Then `std::max( userPropVal, minimum_activation ) <= min_control <= 1`

(1) is enforced at L107 and L116, but (2) is not.